packaging/test_plugin_indexing.py

```diff
@@ -476,7 +471,7 @@ def test_response_to_elasticsearch_404_fulltext__issue_34(self):
                 self.remove_all_jobs_from_delay_queue(admin_session)
                 admin_session.assert_icommand('imkdir -p {0}'.format(collection))
                 f = tempfile.NamedTemporaryFile()
-                f.write( "the quick brown fox"
+                f.write( "the quick brown fox".encode('utf-8')
                         ); f.flush();
```

Comment:
Consider moving the `);` onto the line above. L480/475 looks funky

Reply:
Agreed

Reply:
Please confirm whether new commit addresses the funk in the desired fashion

Reply:
It does.